iib/web/api_v1.py

```diff
@@ -185,6 +185,48 @@ def get_build_logs(request_id):
         return flask.Response(f.read(), mimetype='text/plain')
 
 
+@api_v1.route('/builds/<int:request_id>/related_bundle_images')
+def get_related_bundle_images(request_id):
+    """
+    Retrieve the related bundle images from the bundle CSV for a regenerate-bundle request.
+
+    :param int request_id: the request ID that was passed in through the URL.
+    :rtype: flask.Response
+    :raise NotFound: if the request is not found or there are no logs for the request
```

Comment:
This line and the line below reference "logs" when they should reference related bundle images.